docs/DeveloperGuide.md

```diff
@@ -395,8 +395,29 @@ otherwise)
 
 ### Glossary
 
+* **AddressBook**: A personal management tool used to store contact details, such as names and phone numbers
```

Comment:
Should we avoid references to "AddressBook" from now on as our product is called "ClickConnect"?

I think referencing the address book (separate words) within our application should still be fine though!